CondFormats/SiStripObjects/interface/DeDxCalibration.h

```diff
@@ -0,0 +1,18 @@
+#ifndef DEDXCALIBRATION_H
+#define DEDXCALIBRATION_H
+#include "CondFormats/Serialization/interface/Serializable.h"
+
+#include <vector>
+class DeDxCalibration {
+public:
+  DeDxCalibration();
+  virtual ~DeDxCalibration() {}
+  std::vector<double> thr;
+  std::vector<double> alpha;
+  std::vector<double> sigma;
+  typedef std::pair<uint32_t, unsigned char> ChipId;
+  std::map<ChipId, float> gain;
+
+  COND_SERIALIZABLE;
```

Comment:
in order to manipulate / dump this type of format please apply the changes in `CondCore/Utilities` as per this commit https://github.com/cms-sw/cmssw/commit/760f7be375bb4c1c7626fbc9aea99e83ead1965e

Reply:
Addressed in https://github.com/cms-sw/cmssw/commit/d5037052739eade4f196f6779f9c753c25ba9784